website/sidebars.json

```diff
@@ -5,7 +5,8 @@
       "nos-client/getting-started",
       "nos-client/api",
       "nos-client/implementation-example",
-      "nos-client/features"
+      "nos-client/features",
+      "nos-client/troubleshooting"
```

Comment:
I'm sorry! It's actually going to need the id from the troubleshooting file, which is `installation-troubleshooting` in this case. But I'd suggest renaming the id to `troubleshooting`

I'll clarify this in the readme :)

Reply:
Thank you! Done!